controllers/storagecluster/external_resources.go

```diff
@@ -739,6 +770,23 @@ func (r *StorageClusterReconciler) enableCsiDrivers(availableSCCs []StorageClass
 
 	}
 
+	// Read topology-failure-domain-label and topology-failure-domain-values from external resources and update ConfigMap
+	if instance.Spec.ExternalStorage.Enable {
+		topologyDomainLabel, topologyDomainValues, err := r.getTopologyFailureDomainConfig(instance.UID)
+		if err != nil {
+			r.Log.Error(err, "failed to get topology failure domain config from external resources")
+			return err
+		} else {
+			if topologyDomainLabel != "" && topologyDomainValues != "" {
+				clientConfig.Data["TOPOLOGY_FAILURE_DOMAIN_LABEL"] = topologyDomainLabel
+				clientConfig.Data["TOPOLOGY_FAILURE_DOMAIN_VALUES"] = topologyDomainValues
+
+				r.Log.Info("Updated ocs-client-operator-config with topology failure domain config",
+					"label", topologyDomainLabel, "values", topologyDomainValues)
+			}
```

Comment:
redundant logging, you'd anyways realize this in final config map and don't want to log every reconcile.